ggshield/sca/output/text_handler.py

```diff
@@ -185,3 +186,11 @@ def sca_incident_cve_ids(vulnerability: SCAVulnerability) -> str:
     else:
         cve_str = ", ".join(vulnerability.cve_ids)
     return f"CVE IDs: {cve_str}\n"
+
+
+def sca_incident_identifier(vulnerability: SCAVulnerability) -> str:
+    identifier = vulnerability.identifier
+    # TODO Remove clause when identifier field is in production on backend side (v2.36)
+    if identifier is None:
+        return ""
+    return f"Identifier: {vulnerability.identifier}\n"
```

Comment:
`identifier` instead of `vulnerability.identifier`

Reply:
oups, fixed :)